src/tools/clippy/clippy_utils/Cargo.toml

```
@@ -5,12 +5,12 @@ edition = "2018"
 publish = false
 
 [dependencies]
-if_chain = "1.0.0"
-itertools = "0.9"
+if_chain = "1.0"
+itertools = "0.10"
 regex-syntax = "0.6"
 serde = { version = "1.0", features = ["derive"] }
 unicode-normalization = "0.1"
-rustc-semver="1.1.0"
+rustc-semver= "1.1"
```

```suggestion
rustc-semver = "1.1"
```

---

This change was dropped during rebase:
```suggestion
rustc-semver = "1.1"
```